src/Blorc.OpenIdConnect/Services/UserManager.cs

```diff
@@ -5,14 +5,15 @@
     using System.Linq;
     using System.Text.Json;
     using System.Threading.Tasks;
-    using Blorc.Services;
 
     using Microsoft.AspNetCore.Components;
     using Microsoft.AspNetCore.Components.Authorization;
     using Microsoft.JSInterop;
 
-    public class UserManager : IUserManager
+    public sealed class UserManager : IUserManager, IDisposable
```

Comment:
I think IUserManager should implement IDisposable, not the implementation itself?